moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -1184,40 +1184,40 @@ class MoveGroup::MoveGroupImpl
 }
 }
 
-moveit::planning_interface::MoveGroup::MoveGroup(const std::string &group_name, const boost::shared_ptr<tf::Transformer> &tf, const ros::WallDuration &wait_for_servers)
+moveit::planning_interface::MoveGroupInterface::MoveGroupInterface(const std::string &group_name, const boost::shared_ptr<tf::Transformer> &tf, const ros::WallDuration &wait_for_servers)
```

Comment:
> compilation unit lists the full name of all the symbols it provides.

I'm not sure what that means or what the advantage is. Could you explain a bit more?

I'm referring to putting this at the top:

```
namespace moveit
{
namespace planning_interface
{
```